Index: /Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.module.scss b//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.module.scss
new file mode 100644
--- /dev/null	
+++ b//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.module.scss	
@@ -0,0 +1,77 @@
+.FooterContainer {
+  display: flex;
+  align-items: end;
+  justify-content: center;
+  height: 100vh;
+  background-color: #3C325C;
+
+
+  .FooterContent {
+    gap: 2.5rem;
+    height: 70vh;
+    width: 100%;
+    margin: 0 2rem 2rem;
+    padding: 3rem;
+    border-radius: 2rem;
+    background-color: white;
+
+    .FooterLogo {
+      font-size: 3rem;
+      font-weight: 900;
+      color: #3C325C;
+      width: fit-content;
+    }
+
+    p {
+      font-weight: 300;
+      font-size: 2rem;
+      text-align: left;
+      margin-top: 1rem;
+    }
+
+    .contactContainer {
+      display: flex;
+      flex-direction: column;
+
+
+      h2 {
+        font-family: "Open Sans", serif;
+        font-size: 1rem;
+        margin-left: 1rem;
+
+      }
+
+      .contactIcon {
+        color: #3C325C;
+      }
+
+      .contactEmail {
+        margin-top: 2.2rem;
+        font-family: "Open Sans", serif;
+        display: flex;
+
+
+      }
+
+      .contactPhone {
+        margin-top: 1rem;
+        font-family: "Open Sans", serif;
+        display: flex;
+
+
+      }
+
+      .contactLocation {
+        margin-top: 1rem;
+        font-family: "Open Sans", serif;
+        display: flex;
+
+
+      }
+
+
+    }
+
+
+  }
+}
\ No newline at end of file
Index: /Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.tsx b//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.tsx
new file mode 100644
--- /dev/null	
+++ b//Users/addey/WebstormProjects/dietactics/src/app/components/Homepage/FooterComponent.tsx	
@@ -0,0 +1,52 @@
+"use client";
+import React from "react";
+import styles from "./FooterComponent.module.scss"
+import Link from "next/link";
+import {MdEmail} from "react-icons/md";
+import {FaPhone} from "react-icons/fa6";
+import {FaLocationDot} from "react-icons/fa6";
+import {IconContext} from "react-icons";
+
+
+export default function FooterComponent() {
+    return (<>
+        <footer className={styles.FooterContainer}>
+            <section className={styles.FooterContent}>
+                <div className={styles.contactContainer}>
+                    <Link href={"/"} className={styles.FooterLogo}>Dietactics</Link>
+                    <p>Get started with your <br/>supply chain transformation<br/>today!</p>
+                    <div className={styles.contactEmail}>
+                        <MdEmail className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>info&#64;dietactic.com</h2>
+                    </div>
+                    <div className={styles.contactPhone}>
+                        <FaPhone className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>+1 609-423-5803</h2>
+                    </div>
+                    <div className={styles.contactLocation}>
+                        <FaLocationDot className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>939 W. North Avenue Chicago, IL, 60642</h2>
+                    </div>
+                    <div className={styles.contactLocation}>
+                        <FaLocationDot className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>Hongkong, China</h2>
+                    </div>
+                    <div className={styles.contactLocation}>
+                        <FaLocationDot className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>London, UK</h2>
+                    </div>
+                    <div className={styles.contactLocation}>
+                        <FaLocationDot className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>Istanbul, Turkey</h2>
+                    </div>
+                    <div className={styles.contactLocation}>
+                        <FaLocationDot className={styles.contactIcon} size={"1.5em"}/>
+                        <h2>Dubai, UAE</h2>
+                    </div>
+                </div>
+
+
+            </section>
+        </footer>
+    </>)
+}
Index: /Users/addey/WebstormProjects/dietactics/src/app/components/DeveloperCredit/MadeByRollinBoat.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a//Users/addey/WebstormProjects/dietactics/src/app/components/DeveloperCredit/MadeByRollinBoat.tsx b//Users/addey/WebstormProjects/dietactics/src/app/components/DeveloperCredit/MadeByRollinBoat.tsx
new file mode 100644
--- /dev/null	
+++ b//Users/addey/WebstormProjects/dietactics/src/app/components/DeveloperCredit/MadeByRollinBoat.tsx	
@@ -0,0 +1,22 @@
+import Link from "next/link";
+
+export default function MadeByRollinBoat() {
+    return (
+        <>
+            <section style={{
+                height: "7vh",
+                backgroundColor: "inherit",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+                fontFamily: "'Open Sans', serif",
+                fontWeight: "400",
+                fontSize: ".8rem"
+            }}>
+                <p style={{color: "black"}}>Powered By <Link target="_blank"
+                                                             href={"https://rollinboat.com/"}>Rollin&apos; Boat</Link>
+                </p>
+            </section>
+        </>
+    )
+}
\ No newline at end of file
